<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1221</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPlainTextEdit" name="salida"/>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Agrega Partículas</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="id_label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="velocidad_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Velocidad&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="velocidad_spinBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Agregar</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QPushButton" name="agregarInicio_pushButton">
              <property name="text">
               <string>Inicio</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="agregarFinal_pushButton">
              <property name="text">
               <string>Final</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Destino</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="3" column="3">
             <widget class="QSpinBox" name="destinoX_spinBox">
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="destinoX_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;X&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="destinoY_spinBox">
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="destinoY_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" rowspan="2" colspan="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Origen</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="2">
             <widget class="QLabel" name="origenY_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="origenX_spinBox"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="origenX_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;X&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="origenY_spinBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="id_lineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Mostrar</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QPushButton" name="mostrar_pushButton">
              <property name="text">
               <string>Partículas</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="mostrar_grafo_pushButton">
              <property name="text">
               <string>Grafo</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Colores</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="3" column="0">
             <widget class="QLabel" name="azul_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="verde_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;G&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="rojo_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;R&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="rojo_spinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="verde_spinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="azul_spinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="limpiar_pushButton">
           <property name="text">
            <string>Limpiar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="3">
       <widget class="QTableWidget" name="tabla"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="buscar_lineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Identificador de Particula</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buscar_pushButton">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="mostrarTabla_pushButton">
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="grafica_tab">
     <attribute name="title">
      <string>Página</string>
     </attribute>
     <widget class="QGroupBox" name="grafica_groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1061</width>
        <height>711</height>
       </rect>
      </property>
      <property name="title">
       <string>Veamos las Partículas</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QPushButton" name="dibujar_puntos_pushButton">
         <property name="text">
          <string>Dibujar Puntos</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="limpiar_particulas_pushButton">
         <property name="text">
          <string>Limpiar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QGraphicsView" name="grafica"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="dibujar_lineas_pushButton">
         <property name="text">
          <string>Dibujar Lineas</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="fuerza_bruta_pushButton">
         <property name="text">
          <string>Fuerza Bruta</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuOrdenar">
    <property name="title">
     <string>Ordenar</string>
    </property>
    <addaction name="action_ordenar_id"/>
    <addaction name="action_ordenar_distancia"/>
    <addaction name="action_ordenar_velocidad"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionDijkstra"/>
    <addaction name="actionKruskal"/>
    <addaction name="actionGraham"/>
    <addaction name="actionPrim"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuOrdenar"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="action_ordenar_id">
   <property name="text">
    <string>Por ID</string>
   </property>
  </action>
  <action name="action_ordenar_distancia">
   <property name="text">
    <string>Por Distancia</string>
   </property>
  </action>
  <action name="action_ordenar_velocidad">
   <property name="text">
    <string>Por Velocidad</string>
   </property>
  </action>
  <action name="action_ver_puntos">
   <property name="text">
    <string>Ver Puntos</string>
   </property>
  </action>
  <action name="action_unir_puntos">
   <property name="text">
    <string>Unir Puntos</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="text">
    <string>Dijkstra</string>
   </property>
  </action>
  <action name="actionKruskal">
   <property name="text">
    <string>Kruskal</string>
   </property>
  </action>
  <action name="actionGraham">
   <property name="text">
    <string>Graham</string>
   </property>
  </action>
  <action name="actionPrim">
   <property name="text">
    <string>Prim</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
